### PR TITLE
Add unit converter skeleton

### PR DESCRIPTION
**Pull request by maryx**
Updates README and adds the skeleton for the static unit converter.

### CODE REVIEW

**Comment by sethladd** [comment]
We usually don't commit the `.packages` file. That has specific paths for a user's machine. We do, however, commit the pubspec.lock file for applications (which, this is).

**Comment by sethladd** [comment]
I think we might want to ignore `.idea` folder. cc @devoncarew to confirm this can be ignored.

**Comment by sethladd** [comment]
Actually, just use this file: https://github.com/flutter/oss_project_template/blob/master/.gitignore

**Comment by devoncarew** [comment]
That one looks good. For reference, this one (https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/create/.gitignore.tmpl) is what `flutter create` will create.

**Comment by maryx** [comment]
Thanks for the links! Added .gitignore with all items in those two .gitignores. Seems like there are still some files that could be added (the .gradle, local.properties, and xcconfig)?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+
+Submitting changes:
+1. `git fetch upstream`
+2. `git checkout upstream/master -b name_of_your_branch`
+3. Make changes
+4. `git commit -a -m "<your informative commit message>"`
+5. `git push origin name_of_your_branch`
+
+To send a pull request:
+1. `git pull-request` (if you are using Hub) or go to https://github.com/flutter/udacity-course and click the "Compare & pull request" button
```
Comment (a14n) [inline README.md]: Add a link for [Hub](https://hub.github.com/) ?

Comment (maryx) [reply README.md]: Added.

**Comment by sethladd** [comment]
I think we need to hire .gradle directory inside of static_unit_converter

**Comment by maryx** [comment]
Removed .gradle files. I'm not fully certain that all the files that got committed are required but I did compare the files in the PR to the .gitignores in /static-unit-converter, /android and /ios.

**Comment by sethladd** [comment]
lgtm thanks!

**Pull request merged**